From 592ae0c18e7a90f4153e49339421df9df4ffe247 Mon Sep 17 00:00:00 2001
From: Wenlin Kang <wenlin.kang@windriver.com>
Date: Sun, 1 Dec 2024 23:30:11 -0800
Subject: [PATCH 3/3] debian: fixed lintian error and add support for system
 with ostree

1. debian/lintian: silence lintian error systemd-service-file
2. postinst: add support for system with ostree

Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 debian/changelog | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index d1aa7c13..f65fbef7 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+dracut (105-2) unstable; urgency=medium
+
+  * debian/lintian: silence lintian error systemd-service-file
+  * postinst: add support for ostree system
+
+ -- Wenlin Kang <wenlin.kang@windriver.com>  Sun, 01 Dec 2024 23:26:47 -0800
+
 dracut (105-2) unstable; urgency=medium
 
   * autopkgtest:
-- 
2.42.0

